scripts/etc/init.d/rcS

```diff
@@ -85,6 +85,24 @@ sanitize_inputfile()
     fi
 }
 
+install_files()
+{
+  file_to_read=$1
+  sanitize_inputfile file_to_read
+  while read -r line; do
+    comment=$(echo "$line" | cut -c1)
```

Comment:
The `<` redirects the file contents to the `while` loop. Any output to `stdout` from the `while` loop (notably `cp -rv` output) gets piped to `grep`.

Also `done < grep "-v "^#" "$file_to_read"` doesn't work.